app/controllers/host_controller.rb

```diff
@@ -140,7 +140,7 @@ def edit
       unless @host.supports?(:update) # if host doesn't support editing
         add_flash(_("This Host does not support editing"), :error)
         flash_to_session
-        redirect_to(:action => @lastaction, :id => params[:id])
+        redirect_to(:action => @lastaction, :id => params[:id]) and return
```

Comment:
@DavidResende0 was this a bug in the original PR?

Reply:
Technically yes, but I only encountered it while writing the spec test. Error for reference: `AbstractController::DoubleRenderError:
       Render and/or redirect were called multiple times in this action. Please note that you may only call render OR redirect, and at most once per action. Also note that neither redirect nor render terminate execution of the action, so if you want to exit an action after redirecting, you need to do something like "redirect_to(...) and return".`

Reply:
Prefer simply `return redirect_to` since `and return` is technically doing a boolean comparison and if `redirect_to` returned `nil` or `false` this wouldn't return
```suggestion
        return redirect_to(:action => @lastaction, :id => params[:id])
```

Reply:
I dont' believe we ever do this anywhere else, so I'm confused why it's needed at all